### PR TITLE
Henrik

### PR DESCRIPTION
**Pull request by HenrikSkindhoj**

### CODE REVIEW

**Review by christofferfink — COMMENTED** [review]
Typescript er ikke et framework, men superset af javascript 

Vite er ikke et framwork men et build tool

**Review by christofferfink — APPROVED** [review]
Ser da nice ud 🤿🏊🏽😎

File: report.md [hunk 1]
```diff
@@ -27,6 +27,14 @@
 ---
 
 ## Introduction
+In this project the goals is to design and implement the frontend and backend for a program, and manage the program by implementing the principles of DevOps. This program will be a gambling website that will feature games like poker, roulette and coinflip. 
+
+The frontend will be implemented using the JavaScript framework React, paired with Vite as a build tool, and TypeScript, providing a satisfying user experience. The backend will be made with Java/Kotlin with Quarkus, a powerful framework for building Java applications, paired with PostgreSQL for managing the database. The backend will handle core functionality, like game logic, user authentication, and secure data management via OIDC for token-based authentication.
```
Comment (muItilingualism) [inline report.md:32]: We dont use kotlin. nor OIDC.

Also, mention maven

File: report.md [hunk 2]
```diff
@@ -27,6 +27,14 @@
...
+
+An important part of this project is to ensure the use of DevOps practices, to ensure that all components of the project are developed and deployed with automation and continous integration. Docker will be used for containerization, while GitHub and GitHub Actions will handle version control, automated testing, and deployment. The goal of this approach is to allow for frequent updates and efficient scaling.
+
+This report will go through the design and implementation of the project, as well as how DevOps tools and methods has been applied currently, and how they will be used in the future. The current report is based on the work that has been done up to September 22, 2024.
```
Comment (muItilingualism) [inline report.md:36]: this aint murica

Change date to be either "22nd of September 2024" or "2024-09-22".

**Review by muItilingualism — CHANGES_REQUESTED** [review]
Review

**Pull request merged**